src/common/low_precision_transformations/src/fold_fake_quantize.cpp

```diff
@@ -80,7 +80,23 @@ bool FoldFakeQuantizeTransformation::isConstantOutput(std::shared_ptr<ngraph::No
 }
 
 bool FoldFakeQuantizeTransformation::canBeTransformed(const TransformationContext& context, std::shared_ptr<Node> op) const {
-    return NetworkHelper::isConstantPath(op) || isConstantOutput(op);
+    if (!NetworkHelper::isConstantPath(op) && !isConstantOutput(op)) {
+        return false;
+    }
+
+    const auto fq = ov::as_type_ptr<opset1::FakeQuantize>(op);
```

Comment:
minor: just don't forget to remove: Is it connected with an issue from a 107400 ticket? Should we remove when the issue will be fixed?

Reply:
No, it's connected to the 106215 ticket. In FoldFakeQuantizeTransformation we use a custom folding implementation so reference implementation issues don't connected to these changes.
LPT implementation doesn't support FQ layers per spatial dimensions so I modified `canBeTransformed` method to avoid functional failures in case if we try to fold unsupported FQ using this transformation.